### DISCUSSION TITLE
Support or ignore @connection() or others too

### DISCUSSION THREAD

**Discussion by binary64** [hasura/graphql-engine#7231] [General]
Hi,

In my .graphql files, I have something like:
```graphql
query blah {
  blah @connection('blah') {
    id
  }
}
```

Using Apollo Client 3 on the frontend, this works well. Presumably, the `@connection` is stripped out before being sent to Hasura.

However, in my IDE, I have the ability to execute the document against a local Hasura instance. But this fails with 
```json5
{
      "extensions": {
        "path": "$.selectionSet",
        "code": "validation-failed"
      },
      "message": "directive \"connection\" is not defined in the schema"
    }
```

So I'm stuck: how could this be solved?

**Comment by Franksuda** [comment]
are you fix this issue? I also met this issue recently.
